src/components/UniversalNavbar/FancyAnimatedLogo.scss

```diff
@@ -0,0 +1,81 @@
+@import '../Media/Media.scss';
+
+/**
+   * Region:
+   * Fancy animated scroll behavior
+   */
+@include for-phone-only {
```

Comment:
I suppose if we really wanted to make this css modules, we could have the media query for every rule so it's  not nested, but I didn't want to go through all that just yet and I'm still  on the fence as to whether it's  worth it

Reply:
I can't figure out why this for phone only?

Reply:
Oh... I didn't realize that wasn't happening on any other device, was there a recent change removing the effect on other devices?